### ISSUE TITLE
[ComboboxInline.test.tsx] Temporarily skip snapshot test

### ISSUE THREAD

**Issue by jdanprad0** [laboratoriobridge/bold#902]
### **Describe the bug**

The snapshot test for `ComboboxInline` is unstable and non-deterministic, sometimes it passes, sometimes it fails.  
The issue happens because the `aria-labelledby` attribute contains a dynamic ID that changes between test runs.

### **To Reproduce**

1. Run all tests locally or on CI.
2. Sometimes the snapshot test for `ComboboxInline` will fail due to a different `aria-labelledby` value.

### **Expected behavior**

The test should be stable and always produce the same result, regardless of test run or environment.

### **Screenshots**

<details>
<summary>CI</summary>

![Image](https://github.com/user-attachments/assets/ecabefb8-d874-4033-8a76-b49e2abeb2c1)

![Image](https://github.com/user-attachments/assets/0f7fb8c7-c44f-483d-a9b5-77fda272e9d1)

![Image](https://github.com/user-attachments/assets/d7ea2b3e-10c4-443c-bc89-c79b11b71715)

![Image](https://github.com/user-attachments/assets/4b4b7eba-ccca-4892-86c1-d24ed0195b40)

![Image](https://github.com/user-attachments/assets/1a4065c4-3189-4d0a-8ae7-55adb78e45d2)

![Image](https://github.com/user-attachments/assets/7cd57573-d6dd-42b5-a19b-bb9f10f4aa78)

![Image](https://github.com/user-attachments/assets/824d9997-4412-4f45-9713-dc5dc1b97a90)

![Image](https://github.com/user-attachments/assets/71428800-b89e-42af-b40d-cd448a47de7e)

![Image](https://github.com/user-attachments/assets/5d32b588-91ef-4e80-8c7c-75c9f52c98fc)

![Image](https://github.com/user-attachments/assets/70d19b61-a9c6-4193-af04-6b2715c361f9)

![Image](https://github.com/user-attachments/assets/1944d5fa-5202-41c9-9823-1d9d3976cf41)

![Image](https://github.com/user-attachments/assets/bee99571-8c34-4a19-9634-b13d40683927)

![Image](https://github.com/user-attachments/assets/72990f1b-f358-40fd-a427-6507c4714599)

</details>

<details>
<summary>Locally</summary>

![Image](https://github.com/user-attachments/assets/2cce39a3-aab7-4936-b861-5b0696907f5a)

![Image](https://github.com/user-attachments/assets/a35fe691-4b02-42dd-afa3-9f6fe83a66b0)

</details>

### **Tasks**

Skip snapshot test for ComboboxInline to avoid CI failures.
- [x] renders correcly closed
- [x] renders correcly opened
- [x] renders correcly opened and loading
- [x] renders correcly with custom components correctly

### **Additional context**

The test will be properly fixed and re-enabled in issue #904 .
See the error [here](https://github.com/laboratoriobridge/bold/actions/runs/15473970860/job/43577340305).  
File: [ComboboxInline.test.tsx](https://github.com/laboratoriobridge/bold/blob/main/src/components/Combobox/ComboboxInline.test.tsx).
